### ISSUE TITLE
[XamarinProfiler-Windows]: Accessibility: Color Contrast: MAS1.4.3: "Text" present in graph elements of Roots&Cycles window is not having color contrast ratio of 4.5:1

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#426]
### Pre-Requisite:
1. Install Xamarin Profiler.
2. Launch Profiler through ".mlpd" file.
### Steps to Reproduce

1. Follow the Pre-requisite to launch the xamarin profiler.
2. Select the available instruments to "Allocations" and hit "Start Profiling"button.
3. This will invoke "Allocations" profiling page. Navigate to "Root&Cycles" tab and invoke it.
4. Verify the color contrast ratio of "Text Present" under Graph column

### Expected Behavior
Color contrast ratio should be minimum 4.5:1
### Actual Behavior
Color contrast ratio is less than 4.5:1 i.e, 2.5:1 - 2.9:1
See the screenshots attached
### User Impact
Low vision user will feel difficult in seeing the text
### Screendhots
![3 graph -1](https://user-images.githubusercontent.com/22956130/41704597-4e42101a-7554-11e8-868d-5d9bbea36b81.png)
![3 graph -2](https://user-images.githubusercontent.com/22956130/41704607-534a05ae-7554-11e8-98d6-9ae95055a6d8.png)
![3 graph - 3](https://user-images.githubusercontent.com/22956130/41704613-58b5d144-7554-11e8-9f26-b661745dcb5d.png)

### Environment
Microsoft Windows version: 1803(OS Build 17134.112)
Xamarin Profiler version: 1.6.3.342

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS1.4.3
Xamarin-Profiler-Windows
Wipro
